src/components/decks/DeckListing.vue

```diff
@@ -5,12 +5,38 @@
       placeholder="Filter by title..."
       v-model:search="filterText"
       :is-disabled="isDisabled"></clearable-search>
+    <div
+      v-if="screenWidth < 768"
```

Comment:
Rather than swapping in different elements depending on the screen width, I'd prefer to accomplish this through TailwindCSS to keep the code duplication to a minimum. So in this case the breakpoint you're using is `md`, so you would leave everything that's appropriate for small screens (or everywhere) without a prefix, and then override it with `md:` classes. E.g.:

```javascript
<div class="flex md:block">
```